Description: Change default database dir from ./database to /var/lib/ps3mediaserver/database-$(USER)
Author: Papa Issa DIAKHATE <paissad@gmail.com>
--- a/src/main/java/net/pms/dlna/DLNAMediaDatabase.java
+++ b/src/main/java/net/pms/dlna/DLNAMediaDatabase.java
@@ -69,7 +69,7 @@
 	private final int SIZE_GENRE = 64;
 
 	public DLNAMediaDatabase(String name) {
-		dir = "database";
+		dir = "/var/lib/ps3mediaserver/database-" + System.getProperty("user.name");
 		File fileDir = new File(dir);
 		boolean defaultLocation = fileDir.mkdir() || fileDir.exists();
 		if (defaultLocation) {
